File: server/src/config/env.ts
import * as dotenv from 'dotenv';

dotenv.config();

const config = {
    port: process.env.PORT || 3000,
    appSecret: process.env.APP_SECRET ?? '',
    appId: process.env.APP_ID ?? '',
    signingKey: process.env.SIGNING_KEY ?? '',
    appStoreUrl: process.env.APPSTORE_URL ?? '',
    
};

export default config

File: server-fast/app/config/env.py
import os
from dotenv import load_dotenv

load_dotenv()

config = {
    "APP_ID": os.getenv("APP_ID", ""),
    "APP_SECRET": os.getenv("APP_SECRET", ""),
    "SIGNING_KEY": os.getenv("SIGNING_KEY", ""),
    "APPSTORE_URL": os.getenv("APPSTORE_URL", ""),
    "DATABASE_URL": os.getenv("DATABASE_URL", ""),
}